components/formats-bsd/test/loci/formats/utests/CompressDecompressTest.java

```diff
@@ -44,6 +54,19 @@
  */
 public class CompressDecompressTest {
 
+  /** Calculate the MD5 of a byte array. */
+  public static String md5(byte[] b) {
```

Comment:
Note also that Guava gives us things like `md5().hashBytes(b).toString()` so we may not even really need a helper.